Formula/gcc.rb

```diff
@@ -63,15 +64,27 @@ def version_suffix
     end
   end
 
+  resource "bootstrap_gcc" do
+    url "https://phoenixnap.dl.sourceforge.net/project/gnuada/GNAT_GCC%20Mac%20OS%20X/11.1.0/native/gcc-11.1.0-x86_64-apple-darwin15.pkg"
+    sha256 "d947b5db0576cb62942e5ce61f3ef53fb679f07b1adff7a4c0fa19a5e72a9532"
+  end
+
   def install
+    resource("bootstrap_gcc").stage do
+      system "pkgutil", "--expand-full", "gcc-11.1.0-x86_64-apple-darwin15.pkg", buildpath/"bootstrap_gcc"
+    end
+    bootstrap_gcc_prefix = buildpath/"bootstrap_gcc/gcc-11.1.0-x86_64-apple-darwin15.pkg/Payload"
+    ENV["CC"] = bootstrap_gcc_prefix/"bin/gcc"
+    ENV["CXX"] = bootstrap_gcc_prefix/"bin/g++"
+    ENV.append_to_cflags "--sysroot=#{MacOS.sdk_path}"
```

Comment:
Nope, this shouldn't be necessary

Reply:
`configure` script use `${CC}` as ada compilation driver.